docs/_build/html/.buildinfo

```diff
@@ -0,0 +1,4 @@
+# Sphinx build info version 1
+# This file hashes the configuration used when building these files. When it is not found, a full rebuild will be done.
+config: bfa60070bd1108aa0d4716c7cdb93896
+tags: 645f666f9bcd5a90fca523b33c5a78b7
```

Comment:
The entire build folder has been added to the PR as well and needs to be removed.

Reply:
Sorry about that. I have corrected in https://github.com/bbcmicrobit/micropython/pull/692/commits/849eb3def05ba7abb5e58441cae572f16509d6cd